spaceline-all-the-icons-segments.el

```diff
@@ -886,10 +886,10 @@ available updates then restores the current buffer."
          (text-face `(:height ,(spaceline-all-the-icons--height 0.9) :background ,(face-background default-face) :inherit)))
 
     (let-alist icon-alist
-      (when .height (plist-put icon-face :height (spaceline-all-the-icons--height .height)))
+      (when .height (setq icon-face (append (:height `(spaceline-all-the-icons--height .height)))))
```

Comment:
Here, the backtick should be before `(:height` and the comma before `(spaceline...`
_i.e._
``(append `(:height ,(spaceline...``